test/build/50_ch-image.bats

```diff
@@ -63,6 +63,27 @@ EOF
     [[ $output = *"00_tiny"* ]]
 }
 
+@test 'ch-image reset' {
+   export CH_IMAGE_STORAGE="$BATS_TMPDIR"/reset
+   run ch-image pull alpine
+   run ls "$CH_IMAGE_STORAGE"
+   echo "$output"
+   [[ $status -eq 0 ]]
+   [[ $output = *"dlcache"* ]]
+   [[ $output = *"img"* ]]   
+
+   run ch-image reset
+   ls "$BATS_TMPDIR"
+   echo "$output"
+   [[ $status -eq 0 ]]
+   [[ $output != *"reset"* ]]
+
+   run ch-image reset
+   echo "$output"
+   [[ $status -eq 1 ]]
+   [[ $output = *"$CH_IMAGE_STORAGE not a builder storage"* ]]
```

Comment:
I recommend you also test the result you want directly. E.g. is `$CH_IMAGE_STORAGE` supposed to not exist?